server/planning/agendas_async/agendas_async_service.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+from planning.core.service import BasePlanningAsyncService
+from planning.types import AgendasResourceModel
+from superdesk.resource_fields import ID_FIELD, ITEMS
+from planning.common import set_original_creator
+from apps.auth import get_user_id
+from superdesk import get_resource_service
+from superdesk.errors import SuperdeskApiError
+from superdesk.notification import push_notification
+
+
+class AgendasAsyncService(BasePlanningAsyncService[AgendasResourceModel]):
+    async def _generate_planning_info(self, docs):
```

Comment:
These next 3 are for the REST API and not the on the data layer. Though we don't yet have `on_fetched` mechanism in the new REST API layer. 

Instead, could you please comment these out and I will add this functionality into the base REST API class, and add a TODO-ASYNC comment